drop-transfer/src/ws/server/v2.rs

```diff
@@ -189,8 +189,29 @@ impl<const PING: bool> HandlerLoop<'_, PING> {
             }) = self.jobs.remove(&file)
             {
                 if !task.is_finished() {
+                    let file_id = self
+                        .xfer
+                        .file_by_subpath(&file)
+                        .expect("File should be there sice we have a task registered")
+                        .id();
+
+                    debug!(
+                        self.logger,
+                        "Aborting download job: {}:{file_id}",
+                        self.xfer.id()
+                    );
+
                     task.abort();
 
+                    if let Err(err) = self
+                        .state
+                        .transfer_manager
+                        .incoming_finsh_download(self.xfer.id(), file_id)
+                        .await
```

Comment:
the method is misspelled, it should be `incoming_finish_download()`

Reply:
done